tc-discord-webhooks-server/src/main/java/com/github/playerforcehd/tcdiscordwebhooks/notificator/DiscordNotificator.java

```diff
@@ -214,8 +215,8 @@ public void notifyBuildStarted(@NotNull SRunningBuild sRunningBuild, @NotNull Se
     @Override
     public void notifyBuildSuccessful(@NotNull SRunningBuild sRunningBuild, @NotNull Set<SUser> users) {
         String title = "Build succeeded!";
-        String description = "The build with the ID " + sRunningBuild.getBuildId() + " has succeeded!";
-        String url = this.sBuildServer.getRootUrl() + "/viewLog.html?buildId=" + sRunningBuild.getBuildId();
+        String description = "The build with the ID " + sRunningBuild.getBuildNumber() + " has succeeded!";
+        String url = this.sBuildServer.getRootUrl() + "/viewLog.html?buildId=" + sRunningBuild.getBuildNumber();
```

Comment:
Please continue to use `sRunningBuild.getBuildId();` here.
The URL expects the build id. If the build number is customized in the settings, the URL won't work anymore.